state_type: pacejka_car
input_type: pacejka_car
safety_filter_type: simple_collision_avoider